src/library/scala/Option.scala

```diff
@@ -270,7 +270,7 @@ sealed abstract class Option[+A] extends Product with Serializable {
    *  // Returns None because the partial function doesn't cover the case.
    *  Some("ftp") collect {case "http" => "HTTP"}
    *
-   *  // Returns None because None is passed to the collect method.
+   *  // Returns None because there is no value to pass to the partial function.
```

Comment:
The edit has the advantage over the original of not being wrong. However, it does make me go, What does that mean.  What do you think of:
```
// Returns None because the option is empty: there is no value to pass to the partial function.
```
I'm going dig around in the bottom of the basket of rags for my black t-shirt from high school that reads, "There is no value."